### PR TITLE
anagram: Update json for new "input" policy

### PR DESCRIPTION
**Pull request by Stargator**
#996 Updated the schema for canonical data. So I am updating the exercise's json to reflect the change.

### CODE REVIEW

File: exercises/anagram/canonical-data.json [hunk 1]
```diff
@@ -11,119 +11,153 @@
     {
       "description": "no matches",
       "property": "anagrams",
-      "subject": "diaper",
-      "candidates": ["hello", "world", "zombies", "pants"],
+      "input": {
+        "subject": "diaper",
+        "candidates": ["hello", "world", "zombies", "pants"
+        ]
```
Comment (rpottsoh) [inline exercises/anagram/canonical-data.json]: `]` should be at the end of the previous line.

Comment (Stargator) [reply exercises/anagram/canonical-data.json]: Ah, missed that somehow.

Comment (Stargator) [reply exercises/anagram/canonical-data.json]: @rpottsoh Fixed it!

**Comment by rpottsoh** [comment]
This is probably safe to merge.

**Pull request merged**